code-challenges.js

```diff
@@ -21,48 +21,97 @@ const fruit4 = "kiwi"
 // --------------------1) Create a function that takes in a number and determines if the given number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below.
 
 // Pseudo code:
+// Start by creating a function called checkBoil with a parameter of userTemp.
+// Next, use a series of conditionals to first check if the input arguement is above boiling point, returning if it is.
+// Then, we use an else if statement to do the same for at and below boiling point, and a final catch-all else for bad inputs.
+
+const checkBoil = (userTemp) => {
+    const boilingPoint = 212
+    if(userTemp > boilingPoint) {
+        return(`${userTemp} is above boiling point`)
+    } else if(userTemp === boilingPoint) {
+        return(`${userTemp} is at boiling point`)
+    } else if(userTemp < boilingPoint) {
+        return(`${userTemp} is below boiling point`)
+    } else {
+        return("This is not a valid input")
+    }
+}
 
 const temperature1 = 42
+// console.log(checkBoil(temperature1));
 // Expected output: "42 is below boiling point"
 
 const temperature2 = 350
+// console.log(checkBoil(temperature2));
 // Expected output: "350 is above boiling point"
 
 const temperature3 = 212
+// console.log(checkBoil(temperature3));
 // Expected output: "212 is at boiling point"
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Pseudo code:
+// First, we will create a function that takes in two arrays as parameters, use concat to combine the two inputs together into a new array, and then use a length property to find the total length of the new array.
+
+const arrayTotalLength = (array1, array2) => {
+    newArray = array1.concat(array2)
```

Comment:
Very nice approach! One thing to consider is that declaring your variables allows you to control the scope and avoid unintentionally creating global variables.